ml-deep-dive/svm-task.md

```diff
@@ -0,0 +1,11 @@
+# SVM: Task
+
+1. Find an appropriate dataset for this task
+2. Implement SVM on this dataset and push the results to your public Github repo.
+3. Post results in this [Slack thread](https://techlabs-community.slack.com/archives/G01LXNY9F6J/p1612527408007600) with a few words describing your solution.
```

Comment:
Wrong slack thread, use this one: https://techlabs-community.slack.com/archives/G01LXNY9F6J/p1613226232000500

Reply:
- [ ] Fix Slack thread link